libV2/tasks/CreateCheckpoint.js

```diff
@@ -55,6 +73,25 @@ class CreateCheckpoint extends BaseTask {
         }
     }
 
+    async _getLastCheckpoint() {
+        const resp = await this.withWarp10(async warp10 => warp10.fetch({
+            className: 'utapi.checkpoint.master',
+            labels: {
+                node: warp10.nodeId,
+            },
+            start: 'now',
+            stop: -1,
+        }));
+
+        if (resp.result && (resp.result.length === 0 || resp.result[0] === '' || resp.result[0] === '[]')) {
```

Comment:
I believe we should also return `null` if `resp.result` does not exist:

```suggestion
        if (!resp.result || (resp.result.length === 0 || resp.result[0] === '' || resp.result[0] === '[]')) {
```